docs/server/source/cloud-deployment-templates/workflow.rst

```diff
@@ -38,8 +38,14 @@ and buy an SSL/TLS certificate for the FQDN.
 Things Each Node Operator Must Do
 ---------------------------------
 
-☐ Decide on, or find out some names of the things in your node:
-
+☐ Every MongoDB instance in the cluster must have a unique (one-of-a-kind) name.
+Ask the organization managing your cluster if they have a standard
+way of naming instances in the cluster.
+For example, maybe they assign a unique number to each node,
+so that if you're operating node 12, your MongoDB instance would be named
+``mdb-instance-12``.
+Similarly, other instances must also have unique names in the cluster.
+ 
```

Comment:
Might want to add here:
`This MongoDB name must also be a resolvable name. The scope of how to achieve name resolution across your cluster and federation is beyond the scope of this document.`